pkg/services/control/service.proto

```diff
@@ -75,7 +75,16 @@ message SetNetmapStatusRequest {
     // Set netmap status request body.
     message Body {
         // New storage node status in NeoFS network map.
+        // If status is MAINTENANCE, the node checks whether maintenance is
+        // allowed in the network settings. In case of prohibition, the request
+        // is denied. Otherwise, node switches to local maintenance state. To
+        // force local maintenance, use `force_maintenance` flag.
         NetmapStatus status = 1;
+
+        // MAINTENANCE status validation skip flag. If set, node starts local
+        // maintenance regardless of network settings. The flag MUST NOT be
+        // set for any other status.
+        bool force_maintenance = 2;
```

Comment:
What is the reason for restricting it like this? IMO `force` is nice by itself and can easily be extended in future.

Reply:
As we already noticed, sometimes it is tricky https://github.com/nspcc-dev/neofs-node/issues/1877 . Changing description is easier, but I don't insist.

Reply:
> Changing description is easier

I agree, but if we don't restrict using force flag with other statuses, some client can set it just because he can. If we later specify the forcing behavior of this status in docs, it can break the client behavior.